nasa/sample_app
# Fix #1, update sample app to developer guide

**Describe the contribution**
Updated sample app to follow developer's guide. Added sample table. 

**Testing performed**
1. Build cFS
2. Run GroundSystems
3. Verify 3 commands:
```
EVS Port1 42/1/SAMPLE_APP 3: SAMPLE: NOOP command
EVS Port1 42/1/SAMPLE_APP 4: SAMPLE: RESET command
1980-012-14:03:33.00046 Sample App: Table Value 1: 1 Value 2: 2
```

**Expected behavior changes**
no impact to behavior

**System(s) tested on:**
 - Hardware
 - OS: Ubuntu 18.04.03
 - cFE 6.6, GroundSystem 12.0.09a

**Contributor Info**
Anh Van. NASA Goddard

**Community contributors**
You must attach a signed CLA (required for acceptance) or reference one already submitted
